src/pages/sujeongKim/Main/CommentItem.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const CommentItem = props => {
```

Comment:
```suggestion
const CommentItem =({id, userId, comment}) => {
```

wrap up 시간에 배운 객체 구조분해 할당을 이렇게 적용할 수 있겠습니다!